src/components/routing/ScrollToTop/ScrollToTop.test.tsx

```
@@ -0,0 +1,125 @@
+      </MemoryRouter>
+    )
+
+    expect(scrollToSpy).toHaveBeenCalledWith(0, 0)
+  })
+
+  it('should scroll to top when pathname changes', () => {
+    // Component that triggers navigation
+    function TestNavigator() {
+      const navigate = useNavigate()
+      return (
+        <button onClick={() => { navigate('/new-page'); }} data-testid="navigate-btn">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)